src/global/custom.css

```diff
@@ -0,0 +1,45 @@
+
+@font-face {
+  font-family: 'PoppinsSemiBold';
+  font-style: normal;
+  font-weight: normal;
+  src: url('/public/font/Poppins-SemiBold.ttf');
+}
+
+@font-face {
+  font-family: 'PoppinsRegular';
+  font-style: normal;
+  font-weight: normal;
+  src: url('/public/font/Poppins-Regular.ttf');
+}
+
+@font-face {
+  font-family: 'PoppinsMedium';
+  font-style: normal;
+  font-weight: normal;
+  src: url('/public/font/Poppins-Medium.ttf');
+}
+
+@font-face {
+  font-family: 'PoppinsBold';
+  font-style: normal;
+  font-weight: normal;
+  src: url('/public/font/Poppins-Bold.ttf');
+}
+
+* {
+  font-family: PoppinsRegular !important;
+}
+
+h1, h2, h3, h4, h5, h6 {
+  font-family: PoppinsSemiBold !important;
```

Comment:
Is `!important` required here? Is there a way to override the `font-family` through the Ionic Api?

https://ionicframework.com/docs/theming/advanced#global-variables

Reply:
We can fix this later though.